installation/install.sh

```
@@ -2574,6 +2583,238 @@ install_morph_server(){
+    # License Server install state is independent from the ZStack -u flag:
+    # an existing config.yaml means LS upgrade/reinstall and must preserve its
+    # database password; otherwise this is a new LS deployment.
+    if [ -f "$config_file" ]; then
+        LICENSE_SERVER_EXISTING_INSTALL='y'
+        encrypted=`read_license_server_config_password "$config_file"`
+        LICENSE_SERVER_DB_PASSWORD=`decrypt_license_server_db_password "$encrypted"`
+        return 0
+    fi
+
+    LICENSE_SERVER_DB_PASSWORD="$LICENSE_SERVER_DEFAULT_DB_PASSWORD"
+    LICENSE_SERVER_DB_PASSWORD_ENC=`encrypt_license_server_db_password "$LICENSE_SERVER_DB_PASSWORD"`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)